internal/pkg/services/argus/utils/utils.go

```diff
@@ -51,8 +49,8 @@ func LoadPlanId(planName string, resp *argus.PlansResponse) (*string, error) {
 		availablePlanNames = fmt.Sprintf("%s\n- %s", availablePlanNames, *plan.Name)
 	}
 
-	details := fmt.Sprintf("You provided plan_name %q, which is invalid. Available plan names are: %s", planName, availablePlanNames)
-	return nil, &errors.DSAInvalidPlanError{
+	details := fmt.Sprintf("you provided plan name %q, which is invalid. available plan names are: %s", planName, availablePlanNames)
```

Comment:
This can remain capitalized, it will be shown in a new line after the `Error: ...` message